map/chart_generator.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "indexer/feature_altitude.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include "std/deque.hpp"
+#include "std/vector.hpp"
+
+/// \brief fills uniformAltitudeDataM with altitude data which evenly distributed by
```

Comment:
If so I suggest using "maps". The problem is there's something with name map in C++. If I were "namespace map" I would have a compilation error.